frontend/app/views/spree/shared/_footer.html.erb

```diff
@@ -49,8 +49,8 @@
       </div>
       <div class="row d-none d-xl-flex border-top py-5 text-uppercase footer-spree-menu-links">
 
-        <% if spree_top_level_items_for_menu('spree-all-footer').present? %>
-          <% spree_top_level_items_for_menu('spree-all-footer').each do |parent| %>
+        <% if available_menus.for_footer(I18n.locale).present? %>
```

Comment:
```suggestion
        <% if spree_footer_menu.present? %>
```

let's keep as less ruby code from the templates as possible, let's move this into the navigation helper